src/ImageSharp/PixelFormats/Alpha8.cs

```diff
@@ -97,6 +97,19 @@ public void PackFromRgba32(Rgba32 source)
             this.PackedValue = source.A;
         }
 
+        /// <inheritdoc/>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void PackFromArgb32(Argb32 source) {
```

Comment:
Braces should be on their own line for consistency with the rest of the code base.